.github/ISSUE_TEMPLATE/1-bug.yaml

```diff
@@ -0,0 +1,83 @@
+name: "🐛 Plugin Bug Report"
+description: "Submit a plugin bug report to help us improve"
+title: "🐛 <Plugin>: <Title>"
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: We value your time and effort to submit this bug report. 🙏
+  - type: input
+    id: workspace
+    attributes:
+      label: Plugin Workspace
+      description: "Specify the name of the workspace folder (https://github.com/backstage/community-plugins/tree/main/workspaces) this issue refers to"
```

Comment:
shall we ask for the name of the plugin instead of the name of the workspace? If I am an adopter of a plugin, I won't know anything about the structure of this repository.